src/UrlParser.elm

```diff
@@ -440,18 +442,38 @@ parseString parser pathWithQuery =
       )
 
 
+-- Same as Navigation.Location.
+--
+-- Reproduced here to avoid a dependency on that library, which does not function
+-- outside of the browser.
+
+type alias Location =
+  { href : String
+  , host : String
```

Comment:
Replaced `Navigation.Location` with locally defined, but not exposed, type alias